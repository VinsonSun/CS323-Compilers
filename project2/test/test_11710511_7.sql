struct Node {
   int v;
}

int test7() {
   struct Node node;
   int val = node + 1;
}
