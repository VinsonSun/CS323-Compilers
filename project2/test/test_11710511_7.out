Error type 7 at Line 7: binary operation on non-number variables
